.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,35 @@
+---
+name: '🐛 Bug Report'
+about: Something isn't working
+---
+
+## Overview
+
+...
+
+## Consuming repo
+
+> What repo were you working in when this issue occurred?
+
+...
+
+## Labels
+
+- [ ] Add the `Type: Bug` label to this issue.
+
+### Area
+
+- [ ] Add any relevant `Area: *` labels to this issue
+
+### Scope
+
+- Is this issue related to a specific package?
+
+  - [ ] Tag it with the `Package: *` label.
```

Comment:
I think something like `Package: <package_name>` might be more clear. I had to think about what this meant for a second.